app/models/comment.rb

```diff
@@ -0,0 +1,13 @@
+class Comment < ActiveRecord::Base
+  include PublicActivity::Common
+  belongs_to :repository
+  belongs_to :author, class_name: "User", foreign_key: "user_id"
+
+  validates :title, presence: true
+  validates :body, presence: true
+
+  # Returns true if the user is the author of the comment
+  def author?(user)
```

Comment:
No, tried it without this method and it doesn't work. Is it necessary require or include this helper method?